### PR TITLE
Bxmsdoc 7510 master

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: doc-content/enterprise-only/openshift/rhdm711-authoring-ha-ref.adoc [hunk 1]
```diff
@@ -162,7 +159,7 @@ https://cloud.google.com/container-engine/docs/services/[container-engine docume
 A route is a way to expose a service by giving it an externally reachable hostname such as `www.example.com`. A defined route and the endpoints
 identified by its service can be consumed by a router to provide named connectivity from external clients to your applications. Each route consists
 of a route name, service selector, and (optionally) security configuration. See the
-https://access.redhat.com/documentation/en-us/openshift_enterprise/3.11/html/architecture/core-concepts#architecture-core-concepts-routes[Openshift documentation] for more information.
+https://access.redhat.com/documentation/en-us/openshift_enterprise/3.2/html/architecture/core-concepts#architecture-core-concepts-routes[Openshift documentation] for more information.
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhdm711-authoring-ha-ref.adoc:162]: [link] The link returns 404. The latest version of OCP 3 is 3.11.

s/3.2/3.11

"https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/architecture/core-concepts#architecture-core-concepts-routes"

File: doc-content/enterprise-only/openshift/rhdm711-authoring-ref.adoc [hunk 2]
```diff
@@ -425,3 +427,7 @@ This template requires the following secrets to be installed for the application
 
  * decisioncentral-app-secret
  * kieserver-app-secret
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhdm711-authoring-ref.adoc:430]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhdm711-kieserver-ref.adoc [hunk 3]
```diff
@@ -115,7 +112,7 @@ https://cloud.google.com/container-engine/docs/services/[container-engine docume
 A route is a way to expose a service by giving it an externally reachable hostname such as `www.example.com`. A defined route and the endpoints
 identified by its service can be consumed by a router to provide named connectivity from external clients to your applications. Each route consists
 of a route name, service selector, and (optionally) security configuration. See the
-https://access.redhat.com/documentation/en-us/openshift_enterprise/3.11/html/architecture/core-concepts#architecture-core-concepts-routes[Openshift documentation] for more information.
+https://access.redhat.com/documentation/en-us/openshift_enterprise/3.2/html/architecture/core-concepts#architecture-core-concepts-routes[Openshift documentation] for more information.
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhdm711-kieserver-ref.adoc:115]: [link] The link returns 404. The latest version of OCP 3 is 3.11.

s/3.2/3.11

"https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/architecture/core-concepts#architecture-core-concepts-routes"

File: doc-content/enterprise-only/openshift/rhpam711-authoring-ha-ref.adoc [hunk 4]
```diff
@@ -174,7 +171,7 @@ https://cloud.google.com/container-engine/docs/services/[container-engine docume
 A route is a way to expose a service by giving it an externally reachable hostname such as `www.example.com`. A defined route and the endpoints
 identified by its service can be consumed by a router to provide named connectivity from external clients to your applications. Each route consists
 of a route name, service selector, and (optionally) security configuration. See the
-https://access.redhat.com/documentation/en-us/openshift_enterprise/3.11/html/architecture/core-concepts#architecture-core-concepts-routes[Openshift documentation] for more information.
+https://access.redhat.com/documentation/en-us/openshift_enterprise/3.2/html/architecture/core-concepts#architecture-core-concepts-routes[Openshift documentation] for more information.
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-authoring-ha-ref.adoc:174]: [link] The link returns 404. The latest version of OCP 3 is 3.11.

s/3.2/3.11

"https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/architecture/core-concepts#architecture-core-concepts-routes"

File: doc-content/enterprise-only/openshift/rhpam711-prod-immutable-kieserver-amq-ref.adoc [hunk 5]
```diff
@@ -181,7 +178,7 @@ https://cloud.google.com/container-engine/docs/services/[container-engine docume
 A route is a way to expose a service by giving it an externally reachable hostname such as `www.example.com`. A defined route and the endpoints
 identified by its service can be consumed by a router to provide named connectivity from external clients to your applications. Each route consists
 of a route name, service selector, and (optionally) security configuration. See the
-https://access.redhat.com/documentation/en-us/openshift_enterprise/3.11/html/architecture/core-concepts#architecture-core-concepts-routes[Openshift documentation] for more information.
+https://access.redhat.com/documentation/en-us/openshift_enterprise/3.2/html/architecture/core-concepts#architecture-core-concepts-routes[Openshift documentation] for more information.
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-prod-immutable-kieserver-amq-ref.adoc:181]: [link] The link returns 404. The latest version of OCP 3 is 3.11.

s/3.2/3.11

"https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/architecture/core-concepts#architecture-core-concepts-routes"

File: doc-content/enterprise-only/openshift/rhpam711-prod-immutable-kieserver-ref.adoc [hunk 6]
```diff
@@ -133,7 +130,7 @@ https://cloud.google.com/container-engine/docs/services/[container-engine docume
 A route is a way to expose a service by giving it an externally reachable hostname such as `www.example.com`. A defined route and the endpoints
 identified by its service can be consumed by a router to provide named connectivity from external clients to your applications. Each route consists
 of a route name, service selector, and (optionally) security configuration. See the
-https://access.redhat.com/documentation/en-us/openshift_enterprise/3.11/html/architecture/core-concepts#architecture-core-concepts-routes[Openshift documentation] for more information.
+https://access.redhat.com/documentation/en-us/openshift_enterprise/3.2/html/architecture/core-concepts#architecture-core-concepts-routes[Openshift documentation] for more information.
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-prod-immutable-kieserver-ref.adoc:133]: [link] The link returns 404. The latest version of OCP 3 is 3.11.

s/3.2/3.11

"https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/architecture/core-concepts#architecture-core-concepts-routes"

File: doc-content/enterprise-only/openshift/rhpam711-trial-ephemeral-ref.adoc [hunk 7]
```diff
@@ -116,7 +116,7 @@ https://cloud.google.com/container-engine/docs/services/[container-engine docume
 A route is a way to expose a service by giving it an externally reachable hostname such as `www.example.com`. A defined route and the endpoints
 identified by its service can be consumed by a router to provide named connectivity from external clients to your applications. Each route consists
 of a route name, service selector, and (optionally) security configuration. See the
-https://access.redhat.com/documentation/en-us/openshift_enterprise/3.11/html/architecture/core-concepts#architecture-core-concepts-routes[Openshift documentation] for more information.
+https://access.redhat.com/documentation/en-us/openshift_enterprise/3.2/html/architecture/core-concepts#architecture-core-concepts-routes[Openshift documentation] for more information.
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-trial-ephemeral-ref.adoc:119]: [link] The link returns 404. The latest version of OCP 3 is 3.11.

s/3.2/3.11

"https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/architecture/core-concepts#architecture-core-concepts-routes"

File: doc-content/enterprise-only/openshift/rhpam711-trial-ephemeral-ref.adoc [hunk 8]
```diff
@@ -374,3 +374,8 @@ Http Get on http://localhost:8080/services/rest/server/healthcheck
 ==== Secrets
 
 This template requires the following secrets to be installed for the application to run.
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-trial-ephemeral-ref.adoc:377]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhpam711-prod-ref.adoc [hunk 9]
```diff
@@ -667,3 +655,7 @@ This template requires the following secrets to be installed for the application
  * businesscentral-app-secret
  * smartrouter-app-secret
  * kieserver-app-secret
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-prod-ref.adoc:658]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhpam711-prod-immutable-monitor-ref.adoc [hunk 10]
```diff
@@ -366,3 +362,7 @@ This template requires the following secrets to be installed for the application
 
  * smartrouter-app-secret
  * businesscentral-app-secret
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-prod-immutable-monitor-ref.adoc:365]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhpam711-prod-immutable-kieserver-ref.adoc [hunk 11]
```diff
@@ -397,3 +393,7 @@ more information.
 This template requires the following secrets to be installed for the application to run.
 
  * kieserver-app-secret
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-prod-immutable-kieserver-ref.adoc:396]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhpam711-prod-immutable-kieserver-amq-ref.adoc [hunk 12]
```diff
@@ -496,3 +492,7 @@ This template requires the following secrets to be installed for the application
 
  * kieserver-app-secret
  * broker-app-secret
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-prod-immutable-kieserver-amq-ref.adoc:495]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhpam711-managed-ref.adoc [hunk 13]
```diff
@@ -480,3 +472,7 @@ This template requires the following secrets to be installed for the application
 
  * businesscentral-app-secret
  * kieserver-app-secret
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-managed-ref.adoc:475]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhpam711-kieserver-postgresql-ref.adoc [hunk 14]
```diff
@@ -380,3 +376,7 @@ more information.
 This template requires the following secrets to be installed for the application to run.
 
  * kieserver-app-secret
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-kieserver-postgresql-ref.adoc:379]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhpam711-kieserver-mysql-ref.adoc [hunk 15]
```diff
@@ -379,3 +375,7 @@ more information.
 This template requires the following secrets to be installed for the application to run.
 
  * kieserver-app-secret
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-kieserver-mysql-ref.adoc:378]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhpam711-kieserver-externaldb-ref.adoc [hunk 16]
```diff
@@ -376,3 +372,7 @@ Http Get on http://localhost:8080/services/rest/server/healthcheck
 This template requires the following secrets to be installed for the application to run.
 
  * kieserver-app-secret
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-kieserver-externaldb-ref.adoc:375]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhpam711-authoring-ref.adoc [hunk 17]
```diff
@@ -441,3 +441,7 @@ This template requires the following secrets to be installed for the application
 
  * businesscentral-app-secret
  * kieserver-app-secret
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-authoring-ref.adoc:444]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhpam711-authoring-ha-ref.adoc [hunk 18]
```diff
@@ -586,3 +548,5 @@ Using the eap-service-account in the myproject namespace:
 oc policy add-role-to-user view system:serviceaccount:myproject:eap-service-account -n myproject
 ....
 ====
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-authoring-ha-ref.adoc:551]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhdm711-trial-ephemeral-ref.adoc [hunk 19]
```diff
@@ -374,3 +374,8 @@ Http Get on http://localhost:8080/services/rest/server/healthcheck
 ==== Secrets
 
 This template requires the following secrets to be installed for the application to run.
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhdm711-trial-ephemeral-ref.adoc:377]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhdm711-prod-immutable-kieserver-ref.adoc [hunk 20]
```diff
@@ -336,3 +332,7 @@ Http Get on http://localhost:8080/services/rest/server/healthcheck
 This template requires the following secrets to be installed for the application to run.
 
  * kieserver-app-secret
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhdm711-prod-immutable-kieserver-ref.adoc:335]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhdm711-prod-immutable-kieserver-amq-ref.adoc [hunk 21]
```diff
@@ -419,3 +415,7 @@ This template requires the following secrets to be installed for the application
 
  * kieserver-app-secret
  * broker-app-secret
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhdm711-prod-immutable-kieserver-amq-ref.adoc:418]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhdm711-kieserver-ref.adoc [hunk 22]
```diff
@@ -318,3 +314,7 @@ Http Get on http://localhost:8080/services/rest/server/healthcheck
 This template requires the following secrets to be installed for the application to run.
 
  * kieserver-app-secret
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhdm711-kieserver-ref.adoc:317]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhdm711-authoring-ha-ref.adoc [hunk 23]
```diff
@@ -540,3 +502,5 @@ Using the eap-service-account in the myproject namespace:
 oc policy add-role-to-user view system:serviceaccount:myproject:eap-service-account -n myproject
 ....
 ====
+
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhdm711-authoring-ha-ref.adoc:505]: [style] Additional new lines could be removed but does not impact render.

File: doc-content/enterprise-only/openshift/rhdm711-authoring-ha-ref.adoc [hunk 24]
```diff
@@ -75,7 +75,7 @@ https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/
 |`DECISION_CENTRAL_MEMORY_REQUEST` | -- | Decision Central Container memory request. | 3Gi | True
 |`DECISION_CENTRAL_CPU_REQUEST` | -- | Decision Central Container CPU request. | 1500m | True
 |`KIE_SERVER_MEMORY_LIMIT` | -- | KIE Server Container memory limit. | 2Gi | True
-|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1.5Gi | True
+|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1536Mi | True
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhdm711-authoring-ha-ref.adoc:78]: [technical] Most size values are stated in `Gi`, is there a reason for changing this to `1536Mi`?

File: doc-content/enterprise-only/openshift/rhdm711-authoring-ref.adoc [hunk 25]
```diff
@@ -59,7 +59,7 @@ https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/
 |`DECISION_CENTRAL_MEMORY_REQUEST` | -- | Decision Central Container memory request. | 3Gi | True
 |`DECISION_CENTRAL_CPU_REQUEST` | -- | Decision Central Container CPU request. | 1500m | True
 |`KIE_SERVER_MEMORY_LIMIT` | -- | KIE Server Container memory limit. | 2Gi | True
-|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1.5Gi | True
+|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1536Mi | True
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhdm711-authoring-ref.adoc:62]: [technical] Most size values are stated in `Gi`, is there a reason for changing this to `1536Mi`?

File: doc-content/enterprise-only/openshift/rhdm711-kieserver-ref.adoc [hunk 26]
```diff
@@ -44,7 +44,7 @@ https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/
 |`KIE_SERVER_HTTPS_PASSWORD` | `HTTPS_PASSWORD` | The password for the keystore and certificate. | mykeystorepass | False
 |`KIE_SERVER_BYPASS_AUTH_USER` | `KIE_SERVER_BYPASS_AUTH_USER` | Allows the KIE Server to bypass the authenticated user for task-related operations, for example, queries. (Sets the org.kie.server.bypass.auth.user system property) | false | False
 |`KIE_SERVER_MEMORY_LIMIT` | -- | KIE Server Container memory limit. | 2Gi | True
-|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1.5Gi | True
+|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1536Mi | True
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhdm711-kieserver-ref.adoc:47]: [technical] Most size values are stated in `Gi`, is there a reason for changing this to `1536Mi`?

File: doc-content/enterprise-only/openshift/rhdm711-prod-immutable-kieserver-amq-ref.adoc [hunk 27]
```diff
@@ -49,7 +49,7 @@ https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/
 |`DECISION_CENTRAL_SERVICE` | `WORKBENCH_SERVICE_NAME` | The Service name for the optional Decision Central, where it can be reached, to allow service lookups (for example, maven repo usage), if required. | myapp-rhdmcentr | False
 |`ARTIFACT_DIR` | -- | List of directories from which archives will be copied into the deployment folder. If unspecified, all archives in /target will be copied. | -- | False
 |`KIE_SERVER_MEMORY_LIMIT` | -- | KIE Server Container memory limit. | 2Gi | True
-|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1.5Gi | True
+|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1536Mi | True
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhdm711-prod-immutable-kieserver-amq-ref.adoc:52]: [technical] Most size values are stated in `Gi`, is there a reason for changing this to `1536Mi`?

File: doc-content/enterprise-only/openshift/rhdm711-prod-immutable-kieserver-ref.adoc [hunk 28]
```diff
@@ -50,7 +50,7 @@ https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/
 |`DECISION_CENTRAL_SERVICE` | `WORKBENCH_SERVICE_NAME` | The Service name for the optional Decision Central, where it can be reached, to allow service lookups (for example, maven repo usage), if required. | myapp-rhdmcentr | False
 |`ARTIFACT_DIR` | -- | List of directories from which archives will be copied into the deployment folder. If unspecified, all archives in /target will be copied. | -- | False
 |`KIE_SERVER_MEMORY_LIMIT` | -- | KIE Server Container memory limit. | 2Gi | True
-|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1.5Gi | True
+|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1536Mi | True
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhdm711-prod-immutable-kieserver-ref.adoc:53]: [technical] Most size values are stated in `Gi`, is there a reason for changing this to `1536Mi`?

File: doc-content/enterprise-only/openshift/rhpam711-authoring-ha-ref.adoc [hunk 29]
```diff
@@ -82,7 +82,7 @@ https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/
 |`BUSINESS_CENTRAL_CPU_REQUEST` | -- | Business Central Container CPU Request. | 1500m | True
 |`BUSINESS_CENTRAL_MEMORY_REQUEST` | -- | Business Central Container Memory Request. | 3Gi | True
 |`KIE_SERVER_MEMORY_LIMIT` | -- | KIE Server Container memory limit. | 2Gi | True
-|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1.5Gi | True
+|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1536Mi | True
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-authoring-ha-ref.adoc:85]: [technical] Most size values are stated in `Gi`, is there a reason for changing this to `1536Mi`?

File: doc-content/enterprise-only/openshift/rhpam711-authoring-ref.adoc [hunk 30]
```diff
@@ -63,7 +63,7 @@ https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/
 |`BUSINESS_CENTRAL_CPU_REQUEST` | -- | Business Central Container CPU Request. | 1500m | True
 |`BUSINESS_CENTRAL_MEMORY_REQUEST` | -- | Business Central Container Memory Request. | 3Gi | True
 |`KIE_SERVER_MEMORY_LIMIT` | -- | KIE Server Container memory limit. | 2Gi | True
-|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1.5Gi | True
+|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1536Mi | True
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-authoring-ref.adoc:66]: [technical] Most size values are stated in `Gi`, is there a reason for changing this to `1536Mi`?

File: doc-content/enterprise-only/openshift/rhpam711-kieserver-externaldb-ref.adoc [hunk 31]
```diff
@@ -66,7 +66,7 @@ https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/
 |`KIE_SERVER_BYPASS_AUTH_USER` | `KIE_SERVER_BYPASS_AUTH_USER` | Allows the KIE Server to bypass the authenticated user for task-related operations, for example, queries. (Sets the org.kie.server.bypass.auth.user system property) | false | False
 |`TIMER_SERVICE_DATA_STORE_REFRESH_INTERVAL` | `TIMER_SERVICE_DATA_STORE_REFRESH_INTERVAL` | Sets refresh-interval for the EJB timer database data-store service. | 30000 | False
 |`KIE_SERVER_MEMORY_LIMIT` | -- | KIE Server Container memory limit. | 2Gi | True
-|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory request. | 1.5Gi | True
+|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory request. | 1536Mi | True
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-kieserver-externaldb-ref.adoc:69]: [technical] Most size values are stated in `Gi`, is there a reason for changing this to `1536Mi`?

File: doc-content/enterprise-only/openshift/rhpam711-kieserver-mysql-ref.adoc [hunk 32]
```diff
@@ -53,7 +53,7 @@ https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/
 |`KIE_SERVER_BYPASS_AUTH_USER` | `KIE_SERVER_BYPASS_AUTH_USER` | Allows the KIE Server to bypass the authenticated user for task-related operations, for example, queries. (Sets the org.kie.server.bypass.auth.user system property) | false | False
 |`TIMER_SERVICE_DATA_STORE_REFRESH_INTERVAL` | `TIMER_SERVICE_DATA_STORE_REFRESH_INTERVAL` | Sets refresh-interval for the EJB timer database data-store service. | 30000 | False
 |`KIE_SERVER_MEMORY_LIMIT` | -- | KIE Server Container memory limit. | 2Gi | True
-|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory request. | 1.5Gi | True
+|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory request. | 1536Mi | True
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-kieserver-mysql-ref.adoc:56]: [technical] Most size values are stated in `Gi`, is there a reason for changing this to `1536Mi`?

File: doc-content/enterprise-only/openshift/rhpam711-kieserver-mysql-ref.adoc [hunk 33]
```diff
@@ -112,9 +112,6 @@ https://cloud.google.com/container-engine/docs/services/[container-engine docume
 |8080 | http
 .2+| All the KIE Server web server's ports.
 |8443 | https
-.1+| `${APPLICATION_NAME}-kieserver-ping`
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/rhpam711-kieserver-mysql-ref.adoc:115]: [technical] Some docs only remove `ping` value while others remove `${APPLICATION_NAME}-kieserver-ping` and "The JGroups ping port for clustering."

**Review by csherrar — COMMENTED** [review]
Some questions and a link correction. Started marking each and realized that it is most likely a single change in the script. Otherwise looks good to me.

File: doc-content/enterprise-only/openshift/rhdm711-kieserver-ref.adoc [hunk 34]
```diff
@@ -214,11 +211,10 @@ Http Get on http://localhost:8080/services/rest/server/healthcheck
 |=============
 |Deployments | Name  | Port  | Protocol
 
-.4+| `${APPLICATION_NAME}-kieserver`
+.3+| `${APPLICATION_NAME}-kieserver`
 |jolokia | 8778 | `TCP`
 |http | 8080 | `TCP`
 |https | 8443 | `TCP`
-|ping | 8888 | `TCP`
```
[the remark this answers is not in the document]
Comment (mramendi) [reply doc-content/enterprise-only/openshift/rhdm711-kieserver-ref.adoc:221]: @csherrar no Idea! this just means the engineers removed it from the template.

File: doc-content/enterprise-only/openshift/rhpam711-authoring-ha-ref.adoc (hunk 29 above)
Comment (mramendi) [reply doc-content/enterprise-only/openshift/rhpam711-authoring-ha-ref.adoc:85]: No Idea again - this reflects template changes so these questions should go to engineering

File: doc-content/enterprise-only/openshift/rhpam711-kieserver-mysql-ref.adoc (hunk 33 above)
Comment (mramendi) [reply doc-content/enterprise-only/openshift/rhpam711-kieserver-mysql-ref.adoc:115]: Again not a doc question but a template question

**Pull request merged**